docs/conf.py

```diff
@@ -273,7 +285,7 @@
 texinfo_documents = [
     (
         'index', 'primer3-py', u'primer3-py Documentation',
-        u'Ben Pruitt, Nick Conway; Wyss Institute', 'primer3-py', 'One line description of project.',
+        'Ben Pruitt, Nick Conway; Wyss Institute', 'primer3-py', 'One line description of project.',
```

Comment:
Maybe we need to update `One line description of project`? :)

Reply:
Updated